dropwizard-views-mustache/src/main/java/io/dropwizard/views/mustache/MustacheViewRenderer.java

```diff
@@ -75,7 +81,8 @@ public String getSuffix() {
     }
 
     private MustacheFactory createNewMustacheFactory(Class<? extends View> key) {
-        return new DefaultMustacheFactory(new PerClassMustacheResolver(key));
+        return (fileRoot != null) ? new DefaultMustacheFactory(new FileSystemResolver(fileRoot))
```

Comment:
I think we could make this initialization more terse if we initialize `MustacheResolver` with the ternary operator and pass it to `DefaultMustacheFactory`.
```java
new DefaultMustacheFactory(fileRoot.isPresent() ? new FileSystemResolver(fileRoot.get()) :
            new PerClassMustacheResolver(key))
```